pkg/controllers/awsloadbalancercontroller/deployment.go

```diff
@@ -130,7 +154,7 @@ func desiredDeployment(name, namespace, image, vpcID, clusterName, awsRegion, cr
 									Name:  awsSDKLoadConfigName,
 									Value: "1",
 								},
-							},
+							}, proxy.ReadProxyVarsFromEnv()...),
```

Comment:
Comments here too.  I'm not sure why you would want to take all the proxy var from env for the container.

Reply:
Done.

Reply:
So you just take everything, not filtering for specific env vars that you need.

Reply:
No, [ReadProxyVarsFromEnv](https://pkg.go.dev/github.com/operator-framework/operator-lib@v0.11.0/proxy#ReadProxyVarsFromEnv) takes only the proxy vars as the name suggests.